routeros/resource_certificate_scep_server.go

```diff
@@ -0,0 +1,61 @@
+package routeros
+
+import (
+	"regexp"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceCertificateScepServer() *schema.Resource {
+	resSchema := map[string]*schema.Schema{
+		MetaResourcePath: PropResourcePath("/certificate/scep-server"),
+		MetaId:           PropId(Id),
+
+		"ca_cert": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Name of the CA certificate to use.",
+		},
+		"path": {
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringMatch(regexp.MustCompile("^/scep/"), "path must start with /scep/"),
+			Description:  "HTTP path starting with `/scep/`.",
+		},
+		"next_ca_cert": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "none",
```

Comment:
Thanks for your PR!
I was literally fixing the compatibility of schemas with `Default` values for some attributes today. The time and hexadecimal number comparison functions were changed. In this regard, please: instead of specifying `Default` values, use the functions 
`AlwaysPresentNotUserProvided`/`TimeEquall`/`HexEquall`.

For this